pkg/apis/core/validation/validation.go

```diff
@@ -5506,6 +5506,12 @@ func ValidatePodResize(newPod, oldPod *core.Pod, opts PodValidationOptions) fiel
 		allErrs = append(allErrs, field.Invalid(specPath, newPod.Status.QOSClass, "Pod QOS Class may not change as a result of resizing"))
 	}
 
+	isPodResizeRequestValid := isPodResizeRequestSupported(*oldPod)
+
+	if !isPodResizeRequestValid {
```

Comment:
nit: no need for the variable
```suggestion
	if !isPodResizeRequestSupported(*oldPod) {
```